packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -2311,8 +2309,10 @@ test('mutable liquidity triggers and interest sensitivity', async t => {
     manualTimer.tick();
   }
   await waitForPromisesToSettle();
-  aliceUpdate = await E(aliceNotifier).getUpdateSince();
-  bobUpdate = await E(bobNotifier).getUpdateSince();
-  t.is(aliceUpdate.value.vaultState, Phase.ACTIVE);
+  bobUpdate = await E(bobNotifier).getUpdateSince(bobUpdate.updateCount);
   t.is(bobUpdate.value.vaultState, Phase.LIQUIDATED);
+
+  // No change for Alice
+  aliceUpdate = await E(aliceNotifier).getUpdateSince(); // can't use updateCount because there's no newer update
```

Comment:
`updateCount` is optional. You don't need to justify not including it. Not including it means "I want the current state, even if it hasn't changed."